docs/pages/api-docs/data-grid/grid-cell-params.md

```diff
@@ -14,9 +14,9 @@ import { GridCellParams } from '@material-ui/data-grid';
 
 | Name                                                                                         | Type                                                                              | Description                                                      |
 | :------------------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------- | :--------------------------------------------------------------- |
-| <span class="prop-name">api</span>                                                           | <span class="prop-type">any</span>                                                | GridApi that let you manipulate the grid.                        |
+| <span class="prop-name">api</span>                                                           | <span class="prop-type">GridApi&lt;&gt;</span>                                    | GridApi that let you manipulate the grid.                        |
```

Comment:
`GridApi<>` doesn't seem right. We can open another PR to fix the line below to also check if the array is not empty.

https://github.com/mui-org/material-ui-x/blob/9f336333ef1e537c9ac56b81defa3634f294642a/docs/scripts/buildApi.ts#L81